### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by charlesx2013**
Adding some permission/admin scoping.

### CODE REVIEW

File: app/controllers/api/v1/admin/base_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Api::V1::Admin::BaseController < Api::V1::BaseController
+  # before_filter :authorize_access!
```
Comment (warrenshen) [inline app/controllers/api/v1/admin/base_controller.rb]: Leaving this here?

File: config/routes.rb [hunk 2]
```diff
@@ -4,19 +4,24 @@
...
           end
         end
       end
 
-      resources :announcements, except: [:new, :edit]
+      resources :schools, only: [:index, :show]
+      resources :check_ins, only: [:create, :edit]
+      resources :announcements, only: [:index, :show]
+      resources :users, except: [:new, :edit]
+      resources :semesters, only: [:index, :show]
```
Comment (warrenshen) [inline config/routes.rb]: I don't know how you feel about this but you can add spaces after the commas before the `only:`s and `except:`s for readability.

**Comment by warrenshen** [comment]
So much CUD.

File: app/controllers/api/v1/admin/base_controller.rb (hunk 1 above)
Comment (charlesx2013) [inline app/controllers/api/v1/admin/base_controller.rb]: for later

**Pull request merged**